.github/workflows/remote-copy.yml

```diff
@@ -224,7 +263,28 @@ jobs:
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
           now=$(date +%Y_%m_%d_%H_%M)
+
           mdss -P ${{ vars.PROJECT_CODE }} mkdir -p ${{ vars.TAPE_ROOT_DIR }}/$now
-          mdss -P ${{ vars.PROJECT_CODE }} put -r ${{ needs.copy-to-remote.outputs.files }} ${{ vars.TAPE_ROOT_DIR }}/$now
-          mdss -P ${{ vars.PROJECT_CODE }} put -r ${{ needs.copy-to-remote.outputs.manifests }} ${{ vars.TAPE_ROOT_DIR }}/$now
+          for file in ${{ needs.copy-to-remote.outputs.files }}; do
+            file_relative_to_target=${file#${{ inputs.target }}/}
+            echo "Moving '$file' to ${{ vars.TAPE_ROOT_DIR }}/$now/$file_relative_to_target"
+            dirs_after_target=$(dirname $file_relative_to_target)
+            if [[ "$dirs_after_target" != "." ]]; then
+              mdss -P ${{ vars.PROJECT_CODE }} mkdir -p ${{ vars.TAPE_ROOT_DIR }}/$now/$(dirname $file_relative_to_target)
+            fi
+            mdss -P ${{ vars.PROJECT_CODE }} put -r $file ${{ vars.TAPE_ROOT_DIR }}/$now/$file_relative_to_target
+          done
+
+          for file in ${{ needs.copy-to-remote.outputs.manifests }}; do
+            file_relative_to_target=${file#${{ inputs.target }}/}
+            echo "Moving '$file' to ${{ vars.TAPE_ROOT_DIR }}/$now/$file_relative_to_target"
+            dirs_after_target=$(dirname $file_relative_to_target)
+            if [[ "$dirs_after_target" != "." ]]; then
+              mdss -P ${{ vars.PROJECT_CODE }} mkdir -p ${{ vars.TAPE_ROOT_DIR }}/$now/$(dirname $file_relative_to_target)
+            fi
+            mdss -P ${{ vars.PROJECT_CODE }} put -r $file ${{ vars.TAPE_ROOT_DIR }}/$now/$file_relative_to_target
+          done
```

Comment:
Code is duplicated. Could you just make a single list which is `${{ needs.copy-to-remote.outputs.files }} + ${{ needs.copy-to-remote.outputs.manifests }}` and loop over that?

Reply:
Done in https://github.com/ACCESS-NRI/model-config-inputs/pull/3/commits/3aa4dd1ae5d74e27a50e5e0642d4daaa4e66e403